cilium/datadog_checks/cilium/data/auto_conf.yaml

```diff
@@ -4,7 +4,7 @@
 ## https://docs.datadoghq.com/agent/guide/ad_identifiers/
 #
 ad_identifiers:
-  - cilium-agent
+  - cilium
```

Comment:
hey @ChristineTChen 
I think we can keep both of them, otherwise it would a breaking change for customers who will continue to use `cilium-agent` (`ad_identifiers` is a list).

Reply:
Good catch, i'll add it back in another PR